tests/ctst/world/Zenko.ts

```diff
@@ -304,92 +377,100 @@ export default class Zenko extends World {
 
     /**
      * Creates an assumed role session with a duration of 12 hours.
-     * @Param {boolean} crossAccount - If true, the role will be assumed cross account.
+     * @param {boolean} crossAccount - If true, the role will be assumed cross account.
      * @returns {undefined}
-     * @param crossAccount
      */
-    async prepareAssumeRole(crossAccount: boolean = false) {
+    async prepareAssumeRole(crossAccount = false) {
         this.resetGlobalType();
 
         // Getting default account ID
         const account = await SuperAdmin.getAccount({
-            name: this.parameters.AccountName || Constants.ACCOUNT_NAME,
-        });
-        Zenko.additionalAccountsCredentials[account.name] = {
-            AccessKey: CacheHelper.parameters.AccessKey,
-            SecretKey: CacheHelper.parameters.SecretKey,
+            accountName: this.parameters.AccountName || Constants.ACCOUNT_NAME,
+        }) as Utils.Account['account'];
+        Zenko.additionalAccountsCredentials[account.name as string] = {
+            AccessKey: CacheHelper.parameters?.AccessKey as string,
+            SecretKey: CacheHelper.parameters?.SecretKey as string,
```

Comment:
Same here about the type to be defined above and then:

```suggestion
            AccessKey: CacheHelper.parameters?.AccessKey,
            SecretKey: CacheHelper.parameters?.SecretKey,
```

Reply:
Here, the parameters type is defined in ctst as `parameters : Record<string, unknown> | undefined = undefined;`.
While `Cache.parameters` is indeed set to `this.parameters` in the `init`, the linter doesn't recognize this, so here the casting seems necessary.